### DISCUSSION TITLE
Documentation Overhaul

### DISCUSSION THREAD

**Discussion by ghost** [markqvist/Reticulum#256] [General]
Hello all!
First off, I'm super excited about Reticulum! This project is awesome and the people behind it are awesome.
@markqvist as per your reply on Reticulum I've started to look over the existing manual and suggest changes/take notes on needed updates.
Everywhere I can I have created commits that could be pulled into the main branch. I expect as I go along I will find sections that I don't understand.
Those I will leave to the experienced developers, though I hope to learn enough to be able to contribute more.
My set of changes lives here:
https://github.com/Neo-J-A/Reticulum_Docs_Improvement

For each page (or possibly section) of the manual, I will create a comment in this discussion with my thoughts on it and a pull request with my proposed changes. 
Most changes I suggest will be _formal_ rather than _functional_ until I develop an intimate working knowledge of the project.
They are intended to make reading documentation as easy and fun as possible.
My goal with this is to support the lead developers by reducing documentation load and allowing those with more experience to focus on content over form. If mark or others would rather not see the noise I will refrain from making pull-requests until all the documentation is complete and reviewed.

**Comment by ghost** [comment]
Section 1, whatis.rst
Current page: 
https://markqvist.github.io/Reticulum/manual/whatis.html
My changes: #257 

### Areas that need updates

This is where content changes may need to occur:
- Is every feature under "What does Reticulum Offer?" current?
  - Are the requirements in the first paragraph of "Where can Reticulum be Used?" still true?
- Could use a guide linked into the fourth paragraph.
- Can Reticulum be used under every device listed under "Interface Types and Devices"?

**Comment by ghost** [comment]
# Section 2: Getting Started Fast
Current page: https://markqvist.github.io/Reticulum/manual/gettingstartedfast.html

## Areas that need updates
### Standalone Reticulum Installation
Potentially move the paragraph on installing pip to the end of the section.
This will improve the flow of the section

### Resolving Dependencies and Installation Issues
Refactor this into a subheading under "Standalone installation"? It seems to be mostly pertinent to this section.

### Reticulum-based apps
Mention that Sideband doesn't yet have full feature parity with NomadNet?

### Included Utilities--Participate in development
I would refactor all seven of these headings into its own note, linked back to the "standalone installation" section.
Included utilities are solely relevant for standalone installation/NomadNet.
As are several of the other advanced topics covered in this section.

On the whole I think this piece of documentation is well-written and needs just a little refactoring. I am not in a position to comment on completeness, but as it is a quick-start guide the goal is expediency and not 100% coverage.

**Reply by faragher** [reply]
> ### Reticulum-based apps
> Mention that Sideband doesn't yet have full feature parity with NomadNet?

Is it supposed to? Sideband has a few intended features (like local map) that I don't see as part of NomadNet, and I don't think Sideband is meant to run a propagation node or serve pages.

**Reply by markqvist** [reply]
Nope, it is not. It's intended as a more basic, messaging-oriented app. A nomadnet page-browser would be nice for Android, but I think it is better suited as a separate app.

**Reply by ghost** [reply]
No, it isn't necessarily supposed to be so. This could probably use a mention. 
I went into it expecting feature parity, maybe I was just reading too fast.

**Comment by ghost** [comment]
# Section 3: Using Reticulum On Your System

This section of the docs is very clear. Aside from general updates and currency-checking I think it is good. I notice that in [this commit](https://github.com/markqvist/Reticulum/commit/df184f3e548a5070efc9d686292a6b607896ab79) there's a few updates for the version numbers.
@markqvist does this signify you read through the section for currency?

The top level paragraphs are not clear that Reticulum _can_ be run as a daemon through `rnsd` and as a [system service](https://markqvist.github.io/Reticulum/manual/using.html#reticulum-as-a-system-service)

## Utilities
I love how each utility is compared to a conventional IP-based utility, it helps tie in what the reader already (possibly) understands.
I would like to see the utilities section moved to its own page, as it interrupts the discussion on configuring the Reticulum system.
 
Also, the second paragraph directly under the [header](https://markqvist.github.io/Reticulum/manual/using.html#included-utility-programs) is about the `rnsd` utility.
Does this belong under the [rnsd header](https://markqvist.github.io/Reticulum/manual/using.html#the-rnsd-utility)?
